### PR TITLE
small fixes for prices sync

### PR DESCRIPTION
**Pull request by Pantani**
- Fix priority filter
- Fix CMC parser
- Run schedule in the background
- Add env var to update time

### CODE REVIEW

File: config.yml [hunk 1]
```diff
@@ -34,10 +34,14 @@ market:
   enabled: false
   auth: test
   dex_api: https://dex.binance.org/api
+  dex_quote_update_seconds: 30
   cmc_api: https://pro-api.coinmarketcap.com
   cmc_api_key:
+  cmc_quote_update_seconds: 30
```
Comment (vikmeup) [inline config.yml]: cmc should be 5 minutes at least

File: config.yml [hunk 2]
```diff
@@ -34,10 +34,14 @@ market:
...
+  cmc_rate_update_seconds: 30
   fixer_api: https://data.fixer.io/api
   fixer_key:
+  fixes_rate_update_seconds: 30
```
Comment (vikmeup) [inline config.yml]:
```suggestion
  fixer_rate_update_seconds: 30
```

also it should be ~1 hour? No need to update that often as rates don't change much

File: marketdata/market/dex/dex.go [hunk 3]
```diff
@@ -20,13 +20,14 @@ type Market struct {
 }
 
 func InitMarket() market.Provider {
+	update := viper.GetDuration("market.dex_quote_update_seconds")
 	m := &Market{
 		Market: market.Market{
 			Id:         "dex",
 			Name:       "Binance Dex",
 			URL:        "https://www.binance.org/",
```
Comment (vikmeup) [inline marketdata/market/dex/dex.go]: why do we pass market url here?

File: marketdata/market/dex/dex.go [hunk 4]
```diff
@@ -20,13 +20,14 @@ type Market struct {
 }
 
 func InitMarket() market.Provider {
+	update := viper.GetDuration("market.dex_quote_update_seconds")
 	m := &Market{
 		Market: market.Market{
 			Id:         "dex",
 			Name:       "Binance Dex",
```
Comment (vikmeup) [inline marketdata/market/dex/dex.go]: is name required param?

Comment (Pantani) [reply marketdata/market/dex/dex.go]: no, name and url it's optional. Id and API it's required

File: marketdata/market/dex/dex.go (hunk 3 above)
Comment (Pantani) [reply marketdata/market/dex/dex.go]: I did an API to return all provider, but it's useless and I did remove

**Pull request merged**